### PR TITLE
Sdgamboa/bioc review 1

### PR DESCRIPTION
**Pull request by sdgamboa**
@lwaldron, @LiNk-NY, I've received the first review from Bioc ([here](https://github.com/Bioconductor/Contributions/issues/2791#issuecomment-1282598496)). I made the required/suggested changes and added some more edits (updating the description of the datasets).

>**This Pull request will update the package version to 0.99.1.**

### CODE REVIEW

File: R/MicrobiomeBenchmarkData.R [hunk 1]
```diff
@@ -239,6 +239,6 @@ removeCache <- function(ask = interactive()) {
 #'
 #' @keywords internal
 .getSampleMetadata <- function() {
-    data('sampleMetadata', package = 'MicrobiomeBenchmarkData')
+    utils::data('sampleMetadata', package = 'MicrobiomeBenchmarkData')
     sampleMetadata
```
Comment (LiNk-NY) [inline R/MicrobiomeBenchmarkData.R]: see this section on how to use exported data within the package and avoid putting it in the `.GlobalEnv` 
http://contributions.bioconductor.org/data.html?q=adding#adding-data-to-existing-package

File: R/MicrobiomeBenchmarkData.R [hunk 2]
```diff
@@ -1,4 +1,4 @@
-
+utils::globalVariables('sampleMetadata')
```
Comment (LiNk-NY) [inline R/MicrobiomeBenchmarkData.R]: remove this after you import data with the comment below

**Comment by LiNk-NY** [comment]
Hi Samuel, @sdgamboa 
It looks good. 
- [x] There should only be one `sysdata.rda` in the package and it should be in the R folder.
- [x] `R/sampleMetadata.R` should have the appropriate usage section `@usage data("sampleMetadata", package = "MicrobiomeBenchMarkData")` when `LazyData: false`
- [x] don't forget to get rid of the `data-raw` directory after moving files to `inst/scripts`

**Review by LiNk-NY — CHANGES_REQUESTED** [review]
See the comments in the conversation tab

File: R/MicrobiomeBenchmarkData.R (hunk 1 above)
Comment (sdgamboa) [reply R/MicrobiomeBenchmarkData.R]: Thanks, @LiNk-NY, I created a function with some code as in the example: [e8d7862](https://github.com/waldronlab/MicrobiomeBenchmarkData/pull/8/commits/e8d78622bd1570dbcdeaf89b14e3d30446b1cba3)

**Comment by sdgamboa** [comment]
@LiNk-NY, I think I've done all the requested changes. (I only see a single sysdata.rda file in R, and data-raw has been removed). Is it OK if I merge?

**Review by LiNk-NY — APPROVED** [review]

**Pull request merged**